content/500-support/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx

```diff
@@ -9,7 +9,7 @@ metaDescription: 'Best practice for instantiating PrismaClient with Next.js'
 Lots of users have come across this warning while working with [Next.js](https://nextjs.org/) in development:
 
 ```
-warn(prisma-client) Already 10 Prisma Clients are actively running
+warn(prisma-client) There are already 10 Prisma Client engines actively running.
 ```
```

Comment:
Maybe we can try something like this?
```
warn(prisma-client) Already 10 Prisma Clients are actively running
```
Which was renamed since Prisma Client 3.7.0 to:
```
warn(prisma-client) There are already 10 Prisma Client engines actively running.
```

It might help people looking for the old message